app/controllers/question.rb

```diff
@@ -21,3 +21,23 @@
   @question = Question.find(params[:id])
   erb :'questions/show'
 end
+
+put '/questions/:id' do
+  halt(404, erb(:'404')) unless login?
```

Comment:
need to also have logic for making sure that the current user id (session[user_id]) is the same as the user id of the question posted